npm-shrinkwrap.json

```diff
@@ -1673,9 +1673,9 @@
       "integrity": "sha512-RaE0B+14ToE4l6UqdarKPnXwVDuigfFv+5j9Dze/Nqr23yyuqdNvzcZi3xB+3Agvi5R4EOgAksfv3lXX4vBt9w=="
     },
     "@types/memoizee": {
-      "version": "0.4.4",
-      "resolved": "https://registry.npmjs.org/@types/memoizee/-/memoizee-0.4.4.tgz",
-      "integrity": "sha512-c9+1g6+6vEqcw5UuM0RbfQV0mssmZcoG9+hNC5ptDCsv4G+XJW1Z4pE13wV5zbc9e0+YrDydALBTiD3nWG1a3g=="
+      "version": "0.4.5",
```

Comment:
Builds are failing with the error:
```
** npm-shrinkwrap.json was not deduplicated or not fully committed - FAIL **
** Please run 'npm ci', followed by 'npm dedupe' **
Error: "test-shrinkwrap": Error: /tmp/build/b3c2877b/versioned-source/automation/test-lock-deduplicated.sh failed with exit code 1:
"/tmp/build/b3c2877b/versioned-source/automation/test-lock-deduplicated.sh" [undefined]
```

What I usually do to fix such errors (and always when a switch to a new branch on my computer, where changes to `package.json` are involved) is run:
```
rm -rf node_modules && npm i
```

`npm ci` is not helpful when changes are made to `package.json`, because `npm ci` does not modify the shrinkwrap file as needed. Running `npm install` without first deleting the `node_modules` folder will often (not always) also work correctly, as `npm i` is somewhat influenced by the existing contents of the `node_modules` folder. Combining `npm i` with `dedupe` improves the outcome:

```
npm i && npm dedupe && npm i
```

An easy mistake is to forget to run `npm i` _after_ running `npm dedupe` -- iirc, this one is guaranteed to fail.